avocado/utils/process.py

```diff
@@ -581,6 +578,10 @@ def __init__(self, cmd, verbose=True, allow_output_check=None,
                          of the command result stdout and stderr, by default
                          :data:`avocado.utils.astring.ENCODING`
         :type encoding: str
+        :param logger: User's custom logger, which will be logging the subprocess
+                    outputs. When this variable is not used, the
+                    `avocado.utils.process` logger will be used.
```

Comment:
IMO this should read: "when this *parameter* is not *set*".

Reply:
yes, that is better, I will update that.